com.unity.shadergraph/Editor/Drawing/Blackboard/SGBlackboard.cs

```diff
@@ -89,6 +89,7 @@ BlackboardViewModel ViewModel
         bool m_IsFieldBeingDragged = false;
 
         const int k_DraggedPropertyScrollSpeed = 6;
+        const float k_OriginalScrollerPadding = 12;
```

Comment:
So now with the padding removed in the other PR, we don't need this any more, right?

Reply:
Oh good catch yes this shouldn't be necessary anymore